cdp-agentkit-core/cdp_agentkit_core/actions/social/twitter/account_details.py

```diff
@@ -0,0 +1,33 @@
+
+import tweepy
+from pydantic import BaseModel
+
+ACCOUNT_DETAILS_PROMPT = """
+This tool will return account details for the authenticated user context."""
```

Comment:
Let's be explicit in prompts + docstrings that this is account details for Twitter (X) for more context.

Reply:
roger that!